src/utils.ts

```diff
@@ -10,3 +11,33 @@ export function onWorkspaceRootChange(cb: (rootPath: string) => void): void {
         }
     });
 }
+
+export function checkConfig(): PrettierVSCodeConfig {
+    const config: PrettierVSCodeConfig = workspace.getConfiguration(
+        'prettier'
+    ) as any;
+    if (config.useFlowParser) {
+        window.showWarningMessage(
+            "Option 'useFlowParser' has been deprecated. " +
+                'Use \'parser: "flow"\' instead.'
+        );
+    }
+    if (typeof config.trailingComma === 'boolean') {
+        window.showWarningMessage(
+            "Option 'trailingComma' as a boolean value has been deprecated. " +
+                "Use 'none', 'es5' or 'all' instead."
+        );
+    }
+    return config;
+}
+
+export function allEnabledLanguages(): DocumentSelector {
+    const config = checkConfig();
```

Comment:
Should we check it each time we change active editor ?
btw I'm pretty sure we can now remove those checks. It changed some time ago.